seqr/fixtures/1kg_project.json

```diff
@@ -2829,14 +2829,21 @@
         "institution": "Broad Center for Mendelian Genomics",
         "patient_id": "NA19675"
     }
+}, {
+    "model": "contenttypes.contenttype",
+    "pk": 1,
+    "fields": {
+      "app_label": "seqr",
+      "model": "project-notification"
+    }
```

Comment:
django auto-creates content types but the addition of another databse caused the order of insertion into that  table to change and be less deterministic, so I updated the tests that reference this table to reference explicitly created fixture data with a static key instead of the dynamic data with non-derministic keys